components/locale/src/langid.rs

```diff
@@ -118,6 +118,12 @@ impl LanguageIdentifier {
     }
 }
 
+impl std::fmt::Debug for LanguageIdentifier {
+    fn fmt(&self, f: &mut std::fmt::Formatter) -> std::fmt::Result {
+        f.write_str(&self.to_string())
```

Comment:
Why not `std::fmt::Display::fmt(&self, f)`?  Then you can avoid the intermediate String.  (Not that it matters much since this is only used for debugging)

Reply:
I like it!

Reply:
@dminor can you update to that suggestion? It's a minor one, but establishing a good culture is worth it I think :)

Reply:
Makes sense to me!